CenterForOpenScience/ember-osf-web
# [No ticket] Change exclamation color on collaborative edit warning

- Ticket: [No ticket]
- Feature flag: n/a

## Purpose
- Update color for exclamation icon for collaborative edit warning in draft registrations

## Summary of Changes
- Add style to change color for exclamation icon to change-caution-color

## Screenshots
![Screen Shot 2021-02-19 at 12 13 10 PM](https://user-images.githubusercontent.com/51409893/108538258-9a15e680-72ac-11eb-8fef-86ab531fbfa0.png)


## Side Effects
- `NA`

## QA Notes
This change is only cosmetic. The warning on the right side-bar regarding collaborative editing should be a warning-ish yellowy-orangey color. This warning isn't visible in mobile view
